rules/standard_rules/impossible_travel_login.py

```diff
@@ -87,11 +91,15 @@ def rule(event):
                 deep_get(ipinfo_privacy, "service", default="") != "",
             ]
         )
-    if IS_VPN or IS_PRIVATE_RELAY:
+    # Some satellite networks used during plane travel don't always
+    #   register properly as VPN's, so we have a separate check here.
+    IS_SATELLITE_NETWORK = is_satellite_network(src_ip_enrichments)
```

Comment:
Is the `is_satellite_network` function necessary?  Easier to understand as a single deep_get:

`IS_SATELLITE_NETWORK = deep_get(src_ip_enrichments, "ipinfo_asn", "asn", default="") in SATELLITE_NETWORK_ASNS`